protobufs/stacks/common.proto

```diff
@@ -2,23 +2,45 @@ syntax = "proto3";
 
 package stacks;
 
-/// A type representing a 256-bit integer.
-message Uint256 {
-  // These are the 64 bits of the 256-bit integer from bits 0-63.
-  fixed64 bits_part0 = 1;
-  // These are the 64 bits of the 256-bit integer from bits 64-127.
-  fixed64 bits_part1 = 2;
-  // These are the 64 bits of the 256-bit integer from bits 128-191.
-  fixed64 bits_part2 = 3;
-  // These are the 64 bits of the 256-bit integer from bits 192-255.
-  fixed64 bits_part3 = 4;
+import "crypto/common.proto";
+
+// The id for a transaction on the bitcoin blockchain.
+message BitcoinTxid {
+  crypto.Uint256 txid = 1;
+}
+
+// A bitcoin block hash.
+message BitcoinBlockHash {
+  crypto.Uint256 block_hash = 1;
+}
```

Comment:
Feels a _little_ weird that these are part of the `stacks` package

Reply:
Yeah I was thinking the same thing. When I was opening the PR. Figured I'd move them later. I'll just move them now.

Reply:
Fixed in https://github.com/stacks-network/sbtc/pull/577/commits/291b72e10d107d675ff7b553618a73410c45cb13